systems/summit_user_guide.rst

```diff
@@ -2843,6 +2847,9 @@ on Summit under all compiler families:
 
     module load gdb
 
+Additional information about GDB usage and OLCF-provided builds can be
+found on the `GDB Software Page <https://www.olcf.ornl.gov/software_package/gdb/>`__.
```

Comment:
@jack-morrison @isubsmith similar question as for forge

Reply:
and for valgrind

Reply:
These links are fine for this PR. Updates to this will ultimately come when #5 is addressed, but for the time being I recommend approval.

Reply:
:+1: good enough for me